docs/docs/config/system.md

```diff
@@ -68,6 +68,7 @@ for more information on configuring Bluetooth.
 | `CONFIG_BT`                                 | bool | Enable Bluetooth support                                              |         |
 | `CONFIG_BT_MAX_CONN`                        | int  | Maximum number of simultaneous Bluetooth connections                  | 5       |
 | `CONFIG_BT_MAX_PAIRED`                      | int  | Maximum number of paired Bluetooth devices                            | 5       |
+| `CONFIG_BT_BAS`                             | bool | Enable the Bluetooth BAS (battery reporting service)                  | y       |
```

Comment:
Minor: would be nice to have this in alphabetical order like the rest of the configs here.

Reply:
Ah I wasn't aware that was the convention, all sorted now